aries-site/src/pages/components/checkbox.js

```diff
@@ -24,11 +33,146 @@ const CheckBox = () => (
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Guidance">
+        <SubsectionText>
+          CheckBox can be used for a list of different options that are being
+          provided to a user to select a option as well as being able to select
+          multiple options.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox can be used in a form of toggle. A single option which can
+          indicate something such as on or off. Toggle is prefered to quickly
+          switch between two possible states. Toggles usually provide immediate
+          results in which gives users the ability to control their preference.
+        </SubsectionText>
+        <SubsectionText>
+          CheckBox should not be used if a user can only choose one option from
+          a list of options. In the case that a user should only be able to
+          select one option refer to{' '}
+          <Anchor href="/components/radiobutton">RadioButton</Anchor>
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="About CheckBox" level={3} gap="small">
+        <SubsectionText>
+          There are three different states that the checkbox provides.
+          <BulletedList
+            level={3}
+            items={['Unchecked', 'Checked', 'Indertminate']}
+          />
+          Checked and Unchecked are controlled by the user either clicking on
+          the box or label. Indertminate can be used when a checkbox has a group
+          of selections in which some/ or all can be selected.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="Accessibility" level={3} gap="small">
+        <SubsectionText>
+          CheckBox used be used inside of a FormField to ensure that a label is
+          used as the overall wrapping of the different options being provided
+          by checkboxs. These labels describe the purpose in which the checkbox
+          are serving.
+        </SubsectionText>
+        <SubsectionText>
+          If CheckBox is used outside of Formfield is it still very important to
+          providethe label for each checkbox. These labels should be clear and
+          to the point for the checkbox. These labels should appear to the right
+          of the checkboxes this makes it easier for users to read.
+        </SubsectionText>
+      </Subsection>
+    </ContentSection>
+    <ContentSection>
+      <Subsection name="Variants">
+        <SubsectionText>
+          CheckBox can be used individually, within a group of checkboxes, or as
+          a toggle. A standalone or toggle checkbox indicates that a user is
+          opting-in to the context of the checkbox. Within a group, one or
+          multiple checkboxes can be selected.
+        </SubsectionText>
+      </Subsection>
+      <Subsection name="CheckBox list with hints" level={3}>
+        <SubsectionText>
+          You can add a description that provides your user additional
```

Comment:
Adding a description provides the user with additional information about the context or purpose of the checkboxes.